string type
---
int32 status
FilePath[] filepaths